### PR TITLE
JAMES-3644 Link for SPF and DKIM

### PR DESCRIPTION
**Pull request by chibenwa**
Cc @fiery06

### CODE REVIEW

File: src/homepage/howTo/imap-server.html [hunk 1]
```diff
@@ -222,8 +222,8 @@ <h2><b>Additional features</b></h2>
                     <a href="http://james.apache.org/server/metrics.html">metric display</a>, collect logs in ElasticSearch for a display in Kibana,
                 and much more!</p>
 
-                <p>Also, James offers support for <a href="https://medium.com/linagora-engineering/installing-james-3-0-with-spf-verification-421b26b92f11">SPF</a>
-                    and DKIM standard, which increase the trust external people can get in your mail system.</p>
+                <p>Also, James offers support for <a href="sp.html">SPF</a>
```
Comment (fiery06) [inline src/homepage/howTo/imap-server.html:225]: u miss a "f"

**Review by fiery06 — CHANGES_REQUESTED** [review]

File: src/homepage/howTo/imap-server.html (hunk 1 above)
Comment (chibenwa) [reply src/homepage/howTo/imap-server.html:225]: crap

File: src/homepage/howTo/imap-server.html [hunk 2]
```diff
@@ -222,8 +222,8 @@ <h2><b>Additional features</b></h2>
                     <a href="http://james.apache.org/server/metrics.html">metric display</a>, collect logs in ElasticSearch for a display in Kibana,
                 and much more!</p>
 
-                <p>Also, James offers support for <a href="https://medium.com/linagora-engineering/installing-james-3-0-with-spf-verification-421b26b92f11">SPF</a>
-                    and DKIM standard, which increase the trust external people can get in your mail system.</p>
+                <p>Also, James offers support for <a href="spf.html">SPF</a>
+                    and <a hre="dkim.html">DKIM</a> standard, which increase the trust external people can get in your mail system.</p>
```
Comment (Arsnael) [inline src/homepage/howTo/imap-server.html:226]:
```suggestion
                    and <a hre="dkim.html">DKIM</a> standards, which increase the trust external people can get in your mail system.</p>
```

**Review by fiery06 — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

File: src/homepage/howTo/imap-server.html [hunk 3]
```diff
@@ -222,8 +222,8 @@ <h2><b>Additional features</b></h2>
                     <a href="http://james.apache.org/server/metrics.html">metric display</a>, collect logs in ElasticSearch for a display in Kibana,
                 and much more!</p>
 
-                <p>Also, James offers support for <a href="https://medium.com/linagora-engineering/installing-james-3-0-with-spf-verification-421b26b92f11">SPF</a>
-                    and DKIM standard, which increase the trust external people can get in your mail system.</p>
+                <p>Also, James offers support for <a href="spf.html">SPF</a>
+                    and <a hre="dkim.html">DKIM</a> standards, which increase the trust external people can get in your mail system.</p>
```
Comment (rouazana) [inline src/homepage/howTo/imap-server.html:226]:
```suggestion
                    and <a href="dkim.html">DKIM</a> standards, which increase the trust external people can get in your mail system.</p>
```

**Review by Arsnael — APPROVED** [review]

**Pull request merged**